config/vpc/config.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2021 Upbound Inc.
+*/
+
+package vpc
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
+// Configure adds configurations for vpc group.
+func Configure(p *config.Provider) {
```

Comment:
Hi @turkenf,
We had better understand whether we still need the `SecurityGroupRule.ec2` resource as two separate ingress and egress rules are being introduced. Looking at https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group_rule, Terraform has not deprecated this resource but we may still mark it as deprecated if there are not configurations not covered by these new two resources (for instance, are there any other rule types other than ingress and egress, etc.)?